apps/journeys-admin/src/components/Editor/Slider/JourneyFlow/JourneyFlow.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import Box from '@mui/material/Box'
+import { Meta, StoryObj } from '@storybook/react'
+import { ComponentProps } from 'react'
+
+import { TreeBlock } from '@core/journeys/ui/block'
+import { BlockFields_StepBlock as StepBlock } from '@core/journeys/ui/block/__generated__/BlockFields'
+import { EditorProvider } from '@core/journeys/ui/EditorProvider'
+import { transformer } from '@core/journeys/ui/transformer'
+
+import { journeysAdminConfig } from '../../../../libs/storybook'
+import { blocks, defaultJourney } from '../../data'
+
+import { JourneyFlow } from './JourneyFlow'
+
+const JourneyFlowStory: Meta<typeof JourneyFlow> = {
+  ...journeysAdminConfig,
```

Comment:
I think this looks the same on desktop and mobile so we can use this instead
```suggestion
  ...simpleComponentConfig,
```